cloudbuild.yaml

```diff
@@ -51,8 +51,8 @@ steps:
         --source=. \
         --entry-point=new_dataset \
         --trigger-event-filters="type=google.cloud.storage.object.v1.finalized" \
-        --trigger-event-filters="bucket=${_DATASET_BUCKET}" \
-        --set-env-vars="SCHEMA_BUCKET_NAME=${_SCHEMA_BUCKET_NAME},CONF=cloud-build"
+        --trigger-event-filters="bucket=ons-sds-sandbox-01-europe-west2-dataset-892a" \
```

Comment:
Why the change from variable to hardcoded value?

Reply:
ah thats a mistake thanks for spotting :)